content/en/docs/reference/controllers/built-in.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Built-in controllers
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+This page lists the {{< glossary_tooltip text="controllers" term_id="controller" >}}
+that come as part of Kubernetes itself.
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+Kubernetes comes with a number of built-in controllers that run as part
+of the {{< glossary_tooltip term_id="kube-controller-manager" >}}.
+
+If your cluster is deployed against a cloud service provider, you can
+use the cloud-controller-manager to run additional provider-specific
+controllers such as
+[Route](/docs/concepts/architecture/cloud-controller/#route-controller).
+
+The cloud controller manager provides an abstract API (in Go) that
+allows cloud vendors to plug in their custom implementation.
+
+The built-in {{< glossary_tooltip term_id="kube-scheduler" text="scheduler" >}}
+is itself a specialized controller. The scheduler's purpose is to reconcile the
+desired set of running Pods and match that against the available Nodes,
+optimizing against discovered constraints.
+{{< glossary_tooltip term_id="kubelet" >}} will update the actual state each
+time it starts or stops a scheduled Pod.
+
+Because its work is essential to Kubernetes' operation, the scheduler
+runs separately from the kube-controller-manager. This separation helps
+with control plane performance.
+
+The controllers that run inside kube-controller-manager are:
```

Comment:
The transition on the page from "The controllers that run inside kube-controller-manager are:" to then going to a bold heading of "Controllers for running workloads on Kubernetes" looks really weird.  Check out https://kubernetes.io/docs/setup/ which shows a nice transition from a list of topics (Learning Environment, Production environment) to their sections below the introductory paragraph

Reply:
Thanks for your feedback on this.